resources/lang/en/people.php

```diff
@@ -372,8 +372,8 @@
 
     // Deceased
     'deceased_reminder_title' => 'Anniversary of the death of :name',
-    'deceased_mark_person_deceased' => 'Mark this person as deceased',
-    'deceased_know_date' => 'I know the date this person died',
+    'deceased_mark_person_deceased' => 'Mark :name as deceased',
```

Comment:
This message is used when you create a new contact. It's not possible to add the name of the contact.